plugins/woocommerce/src/Internal/Admin/ProductReviews/Reviews.php

```diff
@@ -133,15 +133,22 @@ public function load_javascript() : void {
 	/**
 	 * Determines if the object is a review or a reply to a review.
 	 *
-	 * @param WP_Comment|array|null $object Object to check.
+	 * @param WP_Comment|mixed $object Object to check.
 	 * @return bool
 	 */
 	protected function is_review_or_reply( $object ) : bool {
-		if ( ! $object instanceof WP_Comment ) {
-			return false;
-		}
 
-		return 'review' === $object->comment_type || 'product' === get_post_type( $object->comment_post_ID );
+		$is_review_or_reply = $object instanceof WP_Comment && ( ( 'review' === $object->comment_type || 'comment' === $object->comment_type ) && 'product' === get_post_type( $object->comment_post_ID ) );
```

Comment:
```suggestion
		$is_review_or_reply = $object instanceof WP_Comment && in_array( $object->comment_type, [ 'review', 'comment' ] ) && 'product' === get_post_type( $object->comment_post_ID );
```

What do you think about changing it to this? That makes the condition slightly simpler IMO.